index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Dire's Portfolio</title>
+    <script src="https://kit.fontawesome.com/b7a7cffe76.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;600&display=swap" rel="stylesheet" >
+</head>
+<body>
+  <main>
+    <header>
+      <div class="header">
+       <h1> <a href="">My Logo</a></h1>
+      <nav><a href=""><i class="fas fa-bars"></i></a></nav>
+      </div> 
+    </header>
+    <section class="container">
+      <div class="intro">
+        <header><h2>Hi! I'M DIRE</h2> 
+          <h2>GLAD TO SEE <br>YOU!</h2>
+        </header> 
+         <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      </div>
+
+    
+       <div class="contact"> 
+          <h4>LET'S CONNECT</h4>
```

Comment:
- [ ] Replace the `h4 `tag with a button or a link. 
`The button (if the template has one) is a link a or button tag, and follow the button styling: Active, Hover and Pressed (see design)`.

- [ ] Align the LET'S CONNECT button with the paragraph at the top.

![Captura de pantalla_2021-09-17_03-03-43](https://user-images.githubusercontent.com/53111464/133747353-3b86bd9b-36cf-4e99-9fc6-afb3efe985cf.png).

- [ ] Kindly use the color values given in the Figma template.